opm/autodiff/ParallelDebugOutput.hpp

```diff
@@ -367,10 +403,10 @@ namespace Opm
                 }
 
                 // write all cell data registered in local state
-		for (const auto& pair : localState_.cellData()) {
```

Comment:
The update of globalReservoirState_ happens in [line 612 of ParallelDebugOutput.hpp](/blattms/opm-simulators/blob/00440ab3449baf915732f8761ef2fa7d48a3486d/opm/autodiff/ParallelDebugOutput.hpp#L612)